source/assets/stylesheets/partials/_blog.scss

```diff
@@ -51,3 +58,100 @@
 .article-preview__content {
   p { margin-bottom: 0.5em; }
 }
+
+.article-title {
+  h1 {
+    text-align: center;
+  }
+}
+
+.article-meta {
+  text-align: center;
+
+  .meta-author {
+    display: inline-block;
+
+   @media only screen and (min-width: 500px) {
+      &::after {
+        content: "–";
+        display: inline;
+        margin: 0 0.5rem;
+      }
+    }
+  }
+}
+
+.article-body {
+  h2,
+  h3,
+  h4 {
+    @include margin($base-spacing null $small-spacing)
+  }
+
+  ul {
+    @include margin(null null $base-spacing $base-spacing * 1.5);
+    list-style-type: disc;
+  }
+
+  ol {
+    @include margin(null null $base-spacing $base-spacing * 1.5);
+    list-style-type: decimal;
+  }
+
+  li {
+    font-size: $base-font-size * 1.2;
+    font-weight: 300;
+
+    @media only screen and (max-width: 500px) {
+      font-size: $base-font-size;
+    }
+  }
```

Comment:
Styling for list items is not displaying properly. Way too big.

![screenshot 2017-09-21 11 44 15](https://user-images.githubusercontent.com/615841/30686928-638e168a-9ec2-11e7-9656-38d814273ac9.png)